javascript-data-model/order-history.js

```diff
@@ -1,66 +1,60 @@
-  ],
-  [
-    {
-      orderPlaced: 'July 19,2020',
-      total: '$44.53',
-      shipTo: 'JS Masher',
-      order: '113-9984268-1280257',
-      deliveredDate: 'Jul 20,2020',
-      bookTitle: 'The Timeless Way of Building',
-      bookAuthor: 'Alexander, Christopher',
-      returnWindowClosingDate: 'Aug 19 2020',
-      price: '$41.33'
-    }
-  ],
-  [
-    {
-      orderPlaced: 'July 4,2020',
-      total: '$17.22',
-      shipTo: 'JS Masher',
-      order: '114-2875557-9059049',
-      deliveredDate: 'Jul 7,2020',
-      itemName: 'Gamecube Controller Adapter, Super Smash Bros Switch Gamcube Adapter for WII U,PC Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
-      returnWindowClosingDate: 'Aug 5, 2020',
-      price: '$15.98'
-    }
-  ],
-  [
-    {
-      orderPlaced: 'July 3,2020',
-      shipTo: 'JS Masher',
-      order: '113-2883177-2648248',
-      items: [
-        {
-          delivereDate: 'July 3,2020',
-          itemName: 'GameCube Controller - Super Smash Bros Edition (Nintendo Switch)',
-          returnWindowClosingDate: 'Aug 4,2020',
-          price: '$94.94'
-        },
-        {
-          deliveredDate: 'July 3, 2020',
-          bookTitle: 'The Art of Sql',
-          bookAuthor: 'Faroult Stephane',
-          returnWindowClosingDate: 'Aug 4,2020',
-          price: '$33.99'
 
-        }
+  {
+    orderPlaced: 'August 4, 2020',
+    total: '$34',
+    shipTo: 'JS Masher',
+    order: '114-3941689-877232',
+    deliveredDate: 'Aug 8, 2020',
+    bookTitle: 'Javscript for Impantient programmers',
+    bookAuthor: 'Rauschmayer, Dr.Axel',
+    returnWindowClosingDate: 'Sep 7, 2020',
+    price: '$31.55'
+  },
+  {
+    orderPlaced: 'July 19,2020',
+    total: '$44.53',
+    shipTo: 'JS Masher',
+    order: '113-9984268-1280257',
+    deliveredDate: 'Jul 20,2020',
+    bookTitle: 'The Timeless Way of Building',
+    bookAuthor: 'Alexander, Christopher',
+    returnWindowClosingDate: 'Aug 19 2020',
+    price: '$41.33'
+  },
+  {
+    orderPlaced: 'July 4,2020',
+    total: '$17.22',
+    shipTo: 'JS Masher',
+    order: '114-2875557-9059049',
+    deliveredDate: 'Jul 7,2020',
+    itemName: 'Gamecube Controller Adapter, Super Smash Bros Switch Gamcube Adapter for WII U,PC Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+    returnWindowClosingDate: 'Aug 5, 2020',
+    price: '$15.98'
+  },
 
-      ]
-    }
+  {
+    orderPlaced: 'July 3,2020',
+    shipTo: 'JS Masher',
+    order: '113-2883177-2648248',
+    items: [
+      {
+        delivereDate: 'July 3,2020',
+        itemName: 'GameCube Controller - Super Smash Bros Edition (Nintendo Switch)',
+        returnWindowClosingDate: 'Aug 4,2020',
+        price: '$94.94'
+      },
+      {
+        deliveredDate: 'July 3, 2020',
+        bookTitle: 'The Art of Sql',
+        bookAuthor: 'Faroult Stephane',
+        returnWindowClosingDate: 'Aug 4,2020',
+        price: '$33.99'
 
-  ]
+      }
+
+    ]
+  }
```

Comment:
Your other objects should match this format.

Reply:
![Screen Shot 2021-01-28 at 6 01 10 PM](https://user-images.githubusercontent.com/69738204/106209531-33147e80-6193-11eb-8193-437b28e2b695.png)
![Screen Shot 2021-01-28 at 6 01 17 PM](https://user-images.githubusercontent.com/69738204/106209540-37d93280-6193-11eb-90aa-28a8b25cedb7.png)